### PR TITLE
add prometheus to app

### PR DESCRIPTION
**Pull request by bartr**

### CODE REVIEW

File: RequestLogger/RequestLogger.cs [hunk 1]
```diff
@@ -53,6 +70,21 @@ public RequestLogger(RequestDelegate next, IOptions<RequestLoggerOptions> option
 
         public static int RequestsPerSecond => RPS.Count > 0 ? RPS[0] : counter;
 
+        /// <summary>
+        /// Return the path and query string if it exists
+        /// </summary>
+        /// <param name="request">HttpRequest</param>
+        /// <returns>string</returns>
+        public static string GetPathAndQuerystring(HttpRequest request)
```
Comment (jofultz) [inline RequestLogger/RequestLogger.cs:78]: I'm wondering if this should be in a different class; maybe a static utils class for this and other static methods that come up but have no home?  It feels like an odd dependency to need the logger to help parse the path and query string.

File: DataAccessLayer/QueryParameters/ValidationError.cs [hunk 2]
```diff
@@ -74,5 +78,96 @@ public static string GetErrorLink(string path)
...
+                subCategory = "Movies";
+                mode = "Query";
+            }
+            else if (path.StartsWith("/api/actors?") || path.StartsWith("/api/actors/?"))
+            {
+                category = "Actors";
+                subCategory = "Actors";
+                mode = "Query";
+            }
+            else if (path.StartsWith("/api/actors/"))
```
Comment (jofultz) [inline DataAccessLayer/QueryParameters/ValidationError.cs:133]: If you move this test to the first actors test then the second test and just look for "/api/actors" and set mode="Query" removing the need for one conditional.

**Review by jofultz — APPROVED** [review]
I had one logic comment, but after I pulled and looked at it in VS I noticed a conditional I had missed that invalidated the comment so I deleted it.

Remaining is one consideration about the location of the GetPathAndQueryString() method.

**Review by jofultz — APPROVED** [review]
I had one logic comment, but after I pulled and looked at it in VS I noticed a conditional I had missed that invalidated the comment so I deleted it.

Remaining is one consideration about the location of the GetPathAndQueryString() method.

File: RequestLogger/RequestLogger.cs (hunk 1 above)
Comment (bartr) [reply RequestLogger/RequestLogger.cs:78]: agree - added a TODO and will move in next PR

**Pull request merged**